src/backend/src/routes/work-packages.routes.ts

```diff
@@ -11,9 +11,6 @@ workPackagesRouter.post(
   '/create',
   intMinZero(body('crId')),
   nonEmptyString(body('name')),
-  intMinZero(body('projectWbsNum.carNumber')),
-  intMinZero(body('projectWbsNum.projectNumber')),
-  intMinZero(body('projectWbsNum.workPackageNumber')),
   isWorkPackageStageOrNone(body('stage')),
   isDate(body('startDate')),
   intMinZero(body('duration')),
```

Comment:
we should also verify that blockedBy is an array, using the isArray() method while you're in here